src/System.Data.Common/ref/Configurations.props

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcoreapp;
-      netfx;
```

Comment:
Can you please regenerate the VS projects/solutions?

Reply:
I'm planning to as a separate commit.  I want that in a place that is easy to re-run and not coupled with a bunch of manual project changes.

Reply:
I went ahead and did it in this PR, let's hope it doesn't reset with merge conflicts too much.